### DISCUSSION TITLE
Is there any way to receive login info.

### DISCUSSION THREAD

**Discussion by wangta69** [openforge/capacitor-game-connect#11] [General]
Is there any way to receive login infomation like id, name... after login to google game?

**Comment by PaulPaulDevelops** [comment]
Yes! Thank you for asking @wangta69. We'll be pushing an update in the next day or so that will return `player_id` and `player_name` from the `signIn` function.
I'll follow up in the this discussion when this has been completed!

**Reply by wangta69** [reply]
I couldn't receive any data after updating v1.0.3.
I tried 3 type of programming

// try 1
CapacitorGameConnect.signIn().then((response) => {
      console.log('login >> response >>', response);
    });

// try 2
    const login = await CapacitorGameConnect.signIn();
    console.log('login >> ', login);

// try 3
const {player_id, player_name} = await CapacitorGameConnect.signIn();
console.log('player_id >> ', player_id, ', player_name >>', player_name);

**Reply by PaulPaulDevelops** [reply]
We appreciate your patience. We realized with our last update we were missing something, so we're still making updates on our end behind the scenes with this to have this fully resolved and available to use. On the next version release, you should be able to use any of these methods you list.
I can follow up with you @wangta69 on this discussion once this is squared away!

**Reply by PaulPaulDevelops** [reply]
Hey @wangta69!

v1.0.4 has been published containing the update in which the `player_name` and `player_id` will return on `signIn`. I installed the new plugin on an example project and verified the update.
Give it a try and let us know what you think! If you have any suggestions of other fields we should return let us know 🙂
